pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -312,15 +312,15 @@ def on_epoch_end(self, epoch, logs=None):
                             self.best = max(self.best_k_models.values())
                         if self.verbose > 0:
                             logging.info(
-                                f'\nEpoch {epoch:05d}: {self.monitor} reached',
-                                f'{current:0.5f} (best {self.best:0.5f}), saving model to',
+                                f'\nEpoch {epoch:05d}: {self.monitor} reached '
```

Comment:
drop space on the end of the line/string and put it on the beginning of next one